src/frontend/komponenter/Stegflyt/StegKnapp.tsx

```diff
@@ -100,9 +110,28 @@ export const StegKnapp: FC<{
         <VStack align="start" gap="4">
             <Feilmelding feil={feilmelding} />
             {behandling.steg === steg && erStegRedigerbart && (
-                <Button variant="primary" size="small" onClick={gåTilNesteSteg} disabled={laster}>
-                    {children}
-                </Button>
+                <>
+                    <StegKnappBekreftelsesModal
+                        modalProps={bekreftelseModal}
+                        gåTilNesteSteg={gåTilNesteSteg}
+                        visModal={visModal}
+                        settVisModal={settVisModal}
+                    />
+                    <Button
+                        variant="primary"
+                        size="small"
+                        onClick={() => {
+                            if (bekreftelseModal) {
+                                settVisModal(true);
+                            } else {
+                                gåTilNesteSteg();
+                            }
```

Comment:
Er vel preferanse, men her kunne vi brukt en `ternary`?

Reply:
Vi bruker if/else ellers i koden for slike flyt. Men vi kan absolut diskutere det i teamet hvis du ønsker :) 
Jeg tenker det i de fleste tilfellene kan være mer lesbart enn 
`bekreftelseModalProps ? settVisModal(true) : gåTilNesteSteg();` Hvis det hadde blitt på 2 ulike rader hadde det kanskje vært litt mer lesbart. 
Vil også trigge lintfeil pga https://eslint.org/docs/latest/rules/no-unused-expressions men som går å konfigurere